packages/rs-dpp/src/state_transition/state_transitions/identity/identity_topup_transition/v0/value_conversion.rs

```diff
@@ -33,10 +34,16 @@ impl<'a> StateTransitionValueConvert<'a> for IdentityTopUpTransitionV0 {
             .map_err(ProtocolError::ValueError)?;
         let asset_lock_proof = AssetLockProof::try_from(raw_asset_lock_proof)?;
 
+        let fee_multiplier = raw_object
+            .get_optional_integer(FEE_MULTIPLIER)
+            .map_err(ProtocolError::ValueError)?
+            .unwrap_or_default();
```

Comment:
Shouldn't bee a DEFAULT constant for multiplier (which has value 2)

Reply:
no, the default is 0, this means that we don't add.

Reply:
I was confused by name.